locale.html

```diff
@@ -173,7 +196,12 @@ <h1>WeekInfoOfLocale ( _loc_ )</h1>
         1. Let _locale_ be _loc_.[[Locale]].
         1. Assert: _locale_ matches the `unicode_locale_id` production.
         1. Let _r_ be a record whose fields are defined by <emu-xref href="#table-locale-weekinfo-record"></emu-xref>, with values based on _locale_.
-        1. Let _fw_ be _loc_.[[FirstDayOfWeek]].
+        1. Let _ca_ be _loc_.[[Calendar]].
+        1. If _ca_ is  *"iso8601"*, then
```

Comment:
This step to handle `"iso8601"` should then be removed, too. Because as currently spec'ed, `ca` takes precedence over `rg`, which contradicts CLDR. The commit message should also be updated to not include "fix #30" to make sure GH doesn't auto-close #30.